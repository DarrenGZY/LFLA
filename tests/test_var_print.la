var c = 2;
var a = 1;
function f()
{
    var b = 5.33;
    print(a);
    print(b);
    print(c);
}

