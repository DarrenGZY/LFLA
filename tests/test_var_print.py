import numpy as np
c=2
def main() :
   a=1
   b=5.33

   print(c)
   print(b)

main()
