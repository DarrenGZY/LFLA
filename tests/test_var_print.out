1
5.33
2
